scripts/src/chartrepomanager/chartrepomanager.py

```diff
@@ -81,11 +92,11 @@ def prepare_chart_tarball_for_release(category, organization, chart, version):
     shutil.copy(path, f".cr-release-packages/{new_chart_file_name}")
     shutil.copy(path, chart_file_name)
 
-def push_chart_release(repository, organization, branch):
+def push_chart_release(repository, organization, branch, commit_hash):
     org, repo = repository.split("/")
     token = os.environ.get("GITHUB_TOKEN")
     print("[INFO] Upload chart using the chart-releaser")
-    out = subprocess.run(["cr", "upload", "-o", org, "-r", repo, "--release-name-template", f"{organization}-"+"{{ .Name }}-{{ .Version }}", "-t", token], capture_output=True)
+    out = subprocess.run(["cr", "upload", "-c", commit_hash, "-o", org, "-r", repo, "--release-name-template", f"{organization}-"+"{{ .Name }}-{{ .Version }}", "-t", token], capture_output=True)
```

Comment:
Is the problem the branch? How is the commit hash fixes that problem? Do you have the man for cr command?